Core/src/org/sleuthkit/autopsy/modules/interestingitems/FilesSetRulePanel.java

```diff
@@ -1016,21 +1083,36 @@ private void fullNameRadioButtonActionPerformed(java.awt.event.ActionEvent evt)
         updateNameTextFieldPrompt();
     }//GEN-LAST:event_fullNameRadioButtonActionPerformed
 
+    private void inclusiveRuleTypeRadioActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_inclusiveRuleTypeRadioActionPerformed
```

Comment:
If these methods are not used remove them.

Reply:
Ok.  Should be fixed.